packages/client/package.json

```diff
@@ -2,18 +2,75 @@
   "name": "filecoin.storage",
   "version": "0.0.0",
   "description": "API client for filecoin.storage",
-  "main": "index.js",
-  "scripts": {
-    "test": "say sorry"
-  },
+  "type": "module",
   "license": "(Apache-2.0 AND MIT)",
   "repository": {
     "type": "git",
     "url": "git+ssh://git@github.com/filecoin-storage/filecoin.storage.git"
   },
-  "author": "",
   "bugs": {
     "url": "https://github.com/filecoin-storage/filecoin.storage/issues"
   },
-  "homepage": "https://github.com/filecoin-storage/filecoin.storage#readme"
+  "homepage": "https://github.com/filecoin-storage/filecoin.storage#readme",
+  "scripts": {
+    "test": "npm-run-all -p -r mock:api test:all",
+    "test:all": "run-s test:web test:cjs test:es",
+    "test:web": "API_PORT=1337 playwright-test test/*.spec.js",
+    "test:cjs": "API_PORT=1337 mocha dist/test/*.spec.cjs --exit",
+    "test:es": "API_PORT=1337 hundreds mocha test/*.spec.js --exit",
+    "mock:api": "smoke -p 1337 --hooks test/mocks/hooks.js test/mocks/api",
+    "build": "run-p build:*",
+    "build:ts": "tsc --build",
+    "build:cjs": "rollup --config --silent rollup.config.js",
+    "coverage": "nyc report --reporter=text-lcov > coverage.lcov && npx codecov",
+    "typedoc": "typedoc --entryPoints src  --out ../../docs/client",
+    "prepare": "npm run build"
+  },
+  "dependencies": {
+    "@ipld/car": "^2.1.0",
+    "@web-std/blob": "^2.1.0",
+    "@web-std/fetch": "^2.0.1"
+  },
+  "devDependencies": {
+    "@ipld/dag-cbor": "^5.0.0",
```

Comment:
this list of devDeps was copied from nft.storage. We don't need all these just yet. I'll whittle it down.